README.md

```diff
@@ -1,10 +1,10 @@
-# To Do List
+# Leaderboard
```

Comment:
- [ ] **[OPTIONAL]** Kindly ensure you use more descriptive branch names going forward. `feature1` gives no hint what the branch is created for.